package.json

```diff
@@ -115,7 +115,7 @@
   },
   "packageManager": "npm@8.19.4",
   "engines": {
-    "node": ">=16 || 14 >=14.18"
+    "node": "14 >=14.18 || >=16"
```

Comment:
Does this matter? Or is there a lint rule/tool reporting this?

Reply:
Unicorn got confused.